### ISSUE TITLE
UI: Improve accessibility experience for overflow menu

### ISSUE THREAD

**Issue by michellezhuogg** [google/shaka-player#1770]
1. If the user opens up the Captions menu and 'Off' is selected, pressing the 'Tab' key will lead to the seek bar, but it should stay on the Captions menu. Same for Resolution, Languages and Picture in Picture.
2. There's no way to hide the Overflow Menu unless the user presses through the page and find the 'More Settings' menu again, which is inconsistent with the visible experience. They should be consistent.

**Comment by ismena** [comment]
@michellezhuogg 
I think #2 has been fixed (ESC closes the menus), what about #1? Have we decided to drop it, I can't remember?

**Comment by joeyparrish** [comment]
I just retested, and both appear to be fixed.  Closing!
